apps/web/src/modules/job/list/pages/index.tsx

```diff
@@ -95,6 +99,15 @@ export default function JobList() {
             gap: '1rem',
           }}
         >
+          {user?.type === UserType.CASTING && (
+            <Button
+              onClick={createPostPage}
+              variant="contained"
+              sx={{ borderRadius: '12px', width: '75%', marginLeft: '1rem' }}
+            >
+              + สร้างโพสต์
+            </Button>
```

Comment:
`+` is too small :joy: Please use [this](https://mui.com/material-ui/react-button/#buttons-with-icons-and-label)